From 3e7faf0075eb7732c4843d79157cbd3f1850f450 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Tue, 16 Jun 2015 03:19:51 +0800
Subject: [PATCH] CHROMIUM: [media] rk3288-vpu: vp8e: increase frame hdr size

Currently, FRAME_HEADER_SIZE is 256, but i've saw
the hdr be larger then 256 when doing screen share.
so we should increase it.

This needs change the define in v4lplugin too.
(change id: Ia6c2271b727218692c4e0b1603d243f32d2f1d77)

BUG=chromium:497324
TEST=screen share through Hangouts

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
Reviewed-on: https://chromium-review.googlesource.com/277398
Reviewed-by: Tomasz Figa <tfiga@chromium.org>

Change-Id: I06e7712420404c43661774e176a7b9333ddc3def
Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
Signed-off-by: Yakir Yang <ykk@rock-chips.com>
---
 drivers/media/platform/rk3288-vpu/rk3288_vpu_hw.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/rk3288-vpu/rk3288_vpu_hw.h b/drivers/media/platform/rk3288-vpu/rk3288_vpu_hw.h
index 4ec286a68dc6..ee5a1466bb05 100644
--- a/drivers/media/platform/rk3288-vpu/rk3288_vpu_hw.h
+++ b/drivers/media/platform/rk3288-vpu/rk3288_vpu_hw.h
@@ -19,7 +19,7 @@
 
 #include <media/videobuf2-core.h>
 
-#define RK3288_HEADER_SIZE		256
+#define RK3288_HEADER_SIZE		1280
 #define RK3288_HW_PARAMS_SIZE		5487
 #define RK3288_RET_PARAMS_SIZE		488
 
-- 
2.35.3

